dashboard/serializers_test.py

```diff
@@ -388,3 +396,52 @@ def test_course_enrollments_serialized_unique(self):
         serialized_program_user = UserProgramSearchSerializer.serialize(self.non_fa_program_enrollment)
         # Number of serialized enrollments should be unaffected
         assert len(serialized_program_user['enrollments']) == serialized_count_before_addition
+
+
+class UserProgramSemesterSerializerEdxTests(TestCase):
+    """
+    Test cases for the serialization of a user's semester enrollments for search results
+    """
+
+    @classmethod
+    def setUpTestData(cls):
+        with mute_signals(post_save):
+            profile = ProfileFactory.create()
+        cls.user = profile.user
+        cls.program = ProgramFactory.create()
+        courses = CourseFactory.create_batch(7, program=cls.program)
+        runs = list()
```

Comment:
If these runs are just relevant to one test we should create the data in that test, so that the input data and the assertions are close together